packages/react/src/elements/components/Banner/BannerAction/styles.scss

```diff
@@ -0,0 +1,41 @@
+@import "~@hig/styles/mixins/colors.scss";
+
+.hig__banner__action {
+  @import "../variables.scss";
+
+  // Remove spacing caused by elements with inline-block
+  display: flex;
+  padding: $action-padding-vertical $action-spacing-horizontal
+    $action-padding-vertical 0;
+  border-right-width: 1px;
+  border-right-style: solid;
+  height: $base-dimension - 10px;
+  align-items: center;
+  margin-left: $action-spacing-horizontal;
+
+  &:first-child {
+    margin-left: 0;
+  }
+
+  &--danger {
+    border-right-color: #333;
```

Comment:
Not sure how much this matters, but we have an app to find the closest HIG color value when given a hex color code. https://ubiquitous-chairs.surge.sh/

Reply:
Yeah, this is a placeholder color. It doesn't match the design at all.